Doc/library/csv.rst

```diff
@@ -443,7 +443,8 @@ read CSV files (assuming they support complex numbers at all).
 .. method:: csvwriter.writerow(row)
 
    Write the *row* parameter to the writer's file object, formatted according to
-   the current dialect.
+   the current dialect, and return the return value of the call to the *write*
```

Comment:
I would prefer to start a new sentence: "the current dialect. Return ...".

Maybe even put it in a new paragraph (I didn't check the style of other functions in csv doc).